plugins/builds/steps/logs.js

```diff
@@ -160,21 +160,13 @@ module.exports = config => ({
             }
         },
         handler: (req, reply) => {
-            const factory = req.server.app.buildFactory;
+            const factory = req.server.app.stepFactory;
```

Comment:
This should be renamed to `stepFactory` like `plugins/builds/get.js` for refactor.

Reply:
I fixed